### PR TITLE
[javascript/en] Added for/in loop JavaScript

### PR DESCRIPTION
**Pull request by raphaelbn**
For/In loop JavaScript

### CODE REVIEW

**Comment by adambrenecki** [comment]
Hi,

Thanks for the contribution!

IMO, it'd be a good idea to at least mention the need for `hasOwnProperty`, since that's a pretty big caveat for people used to `for ... in` in languages like Python.

**Comment by raphaelbn** [comment]
Hi @adambrenecki,

Thanks for your attention!

I will update with yours suggestion and commit.

**Comment by raphaelbn** [comment]
Hi @adambrenecki, I've already made the PR with your suggestions.

File: javascript.html.markdown [hunk 1]
```diff
@@ -218,6 +218,26 @@ for (var i = 0; i < 5; i++){
...
+for (var x in person) {
+    description += person[x] + " ";
+}
+
+//If only want to consider properties attached to the object itself, 
+//and not its prototypes use hasOwnProperty() check
+var description = "";
+var person = {fname:"Paul", lname:"Ken", age:18}; 
+for (var x in person) {
+    if( person.hasOwnProperty( x ) ) {
```
Comment (vendethiel) [inline javascript.html.markdown]: please fix the code style

**Comment by raphaelbn** [comment]
Is it ok now, @vendethiel ?

**Comment by levibostian** [comment]
Thanks!

**Pull request merged**